vtkpytools/barfiletools/data.py

```diff
@@ -240,8 +243,12 @@ def sampleDataBlockProfile(dataBlock, line_walldists, pointid=None,
     else:
         cutterout = vCutter(wall, cutterobj)
         if cutterout.points.shape[0] != 1:
-            raise RuntimeError('vCutter resulted in {:d} points instead of 1.'.format(
-                cutterout.points.shape[0]))
+            if choosePoint == None:
+                raise RuntimeError('vCutter resulted in {:d} points instead of 1.'.format(
```

Comment:
Should probably add a suggestion to the error message to give a `choosePoint` function.